samples/cpp/benchmark_app/benchmark_app.hpp

```diff
@@ -129,6 +129,10 @@ static const char report_type_message[] =
 // @brief message for report_folder option
 static const char report_folder_message[] = "Optional. Path to a folder where statistics report is stored.";
 
+// @brief message for json_stats option
+static const char json_stats_message[] = "Optional. Enables JSON-based statistics output (by default reporting system "
+                                         "will use CSV format). Should be used together with -report_folder option.";
```

Comment:
```suggestion
// @brief message for json_stats option
static const char json_stats_message[] = "Optional. Enables JSON-based statistics output (by default reporting system "
                                         "will use CSV format). Should be used together with -report_type option.";
```
`-report_type` flag enables output, not report folder